front/style/pages/_appel-a-engagement.scss

```diff
@@ -116,7 +116,7 @@
             .space--0-50 {
                 padding: 0;
                 @media (min-width: $mobile) {
-                    padding: 0 50px;
+                    padding: 0 40px;
```

Comment:
for being consistent : `.space--0-50` -> `.space--0-40` ? 😃

Reply:
oh yes